### ISSUE TITLE
Icon Request: fa-internetarchive-square

### ISSUE THREAD

**Issue by soycode** [FortAwesome/Font-Awesome#8359]
https://archive.org/

![Archive.org logo](http://blog.archive.org/wp-content/uploads/2013/04/Internetlogo.jpg)

(Though probably without the text around it.)

**Comment by smajl** [comment]
[sarcasm] We sure desperately need several icons of a roman temple... [/sarcasm]
http://fortawesome.github.io/Font-Awesome/icon/university/

**Comment by soycode** [comment]
My apologies for missing that, thanks for pointing it out.

**Comment by tagliala** [comment]
Sorry, I need to reopen this.

Logos should be implemented as they are

**Comment by smajl** [comment]
@tagliala: That's absolutely true. You can't just mangle logotypes. But... do we really want Font-Awesome to include more and more logo-bloatware of every other web page or service? I would more likely see this implemented https://github.com/FortAwesome/Font-Awesome/issues/1582 than other ten more web company logos which I will never use...

**Comment by tagliala** [comment]
@smajl I'm just reopening: if this request will be popular, that logo could be included.

About the other request, it is one of the most popular requests in the web application category so there are good chances of being included in the future

**Comment by soycode** [comment]
One other thought - a service being relatively more "open" (like Archive) is perhaps a point in favor of inclusion (as is its relative durability, been around since 1996), though I agree popularity is key (and admittedly nobody else has asked for this yet, so we'll just have to wait and see).

**Comment by tecknicaltom** [comment]
archive.org logo posted as an SVG in #9861

**Comment by tagliala** [comment]
Moving to #9861
